multimap/src/main/java/org/infinispan/multimap/impl/EmbeddedMultimapCache.java

```diff
@@ -188,7 +189,7 @@ private Boolean containsEntryInternal(V value) {
    }
 
    private Long sizeInternal() {
-      return cache.values().parallelStream().mapToLong(Bucket::size).sum();
+      return cache.entrySet().parallelStream().mapToLong(entry -> entry.getValue().size()).sum();
```

Comment:
Is this change really needed? I would expect values to work properly.

Reply:
It's not really needed, I was just thinking it's less work if we have a single "map" operation instead of mapping to the value (done inside `ValuesCacheCollection.stream()`) and then mapping the value to the bucket size.